usr/share/lib/img_proof/tests/SLES/SAP/test_sles_sap_hardened.py

```diff
@@ -0,0 +1,60 @@
+import os
```

Comment:
This seems to be a duplication of usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py. If so only test_sles_hardened.py is needed. The sles tests are also run for SAP.

Reply:
The only difference is the profile:

`oscap_profile = "pcs-hardening-sap` vs. `oscap_profile = "pcs-hardening`

Reply:
Ah, didn't notice that. We could break down the components into fixtures so they can be re-used.

Reply:
If there's an sap profile does that mean the default profile is not intended to run against SAP? By adding test_sles_hardened to the base sles test suite we would be testing SAP images for "pcs-hardening" and "pcs-hardening-sap".

Reply:
> If there's an sap profile does that mean the default profile is not intended to run against SAP? By adding test_sles_hardened to the base sles test suite we would be testing SAP images for "pcs-hardening" and "pcs-hardening-sap".

The SAP image will be tested only with the SAP profile.

Reply:
Okay, that makes it easier. We'd only need one test and that would have an "is_sap" check to determine what profile to use.

Reply:
Fixed I guess.